src/OrbitGl/App.cpp

```diff
@@ -1114,6 +1114,14 @@ orbit_base::Future<ErrorMessageOr<void>> OrbitApp::LoadModule(const ModuleData*
 
 orbit_base::Future<ErrorMessageOr<void>> OrbitApp::LoadModule(const std::string& module_path,
                                                               const std::string& build_id) {
+  const ModuleData* module_data = GetModuleByPath(module_path);
+
+  if (module_data == nullptr) {
+    return {ErrorMessage{absl::StrFormat("Module \"%s\" was not found", module_path)}};
```

Comment:
I wasn't aware that this simplification is possible
`Future<ErrorMessageOr<void>>{ErrorMessage{}}` to `{ErrorMessage{}}`
I like it. Thank you, I learned something!

Reply:
Well it's horrible in some way. The problem is that two implicit conversion need to happen here.

I'm going to introduce a specialization for Future<ErrorMessageOr<T>>. This will also get an additional constructor for ErrorMessage.